tests/testdata/kusto/kusto_queries_mstic_ia.yaml

```diff
@@ -101,7 +101,7 @@ sources:
     description: Lists all processes with a command line containing a string
     metadata:
       data_families: [scrubbeddata]
-      cluster: https://wcdscrubbedservice.kusto.windows.net
+      cluster: https://mstic.kusto.windows.net
```

Comment:
Shall we use msticpy for the cluster name here, just in case one day we have a mstic cluster for real. Not a big issue just a thought.

Reply:
Not sure it's a big deal - I don't think there and this doesn't give anyone access. Just thought I'd make it sound a bit more generic